src/components/announcements/IntroProjectTags.tsx

```diff
@@ -1,10 +1,21 @@
+import { Trans } from '@lingui/macro'
+import ProjectTagsRow from 'components/ProjectTagsRow'
+import { projectTagOptions } from 'models/project-tags'
+
 export default function IntroProjectTags() {
   return (
     <div>
-      <h1>Introducing project tags</h1>
-      Project tags are fun and great and fun
+      <h1>
```

Comment:
**suggestion** Semantically speaking, h1 will conflict with the existing h1 for the project title. Prolly best to just keep this as a `span` and set the desired font size, wdyt?

Reply:
Ah great call. Agree with not using h1, but a h3/4 maybe better than a span?